3] guessing game 2/main.py

```diff
@@ -0,0 +1,39 @@
+print("Mysli na číslo v rozmezí 1 - 1000 a já se pokusím jej uhodnout.")
+
+#ověření švindlíře
+my_number = int(input("Sem zadej své číslo: "))
+
+#výchozí hodnoty
+min = 0
+max = 1000
+
+#zacyklení hádání
+x = 0
+
+while x < 10:
```

Comment:
Tohle je pomerne komplikovane reseni. Ty tady vubec nemas pracovat s 10 pokusy jako s kontrolni promenou v tom loopu. To, ze ten pocitace to uhadne maximalne do 10 pokusu je dano tim, ze kdyz ten algoritmus spravne napsany, tak bezi v logaritmicke komplexite a log(1000) je 10. S tim ale ty vubec nemas pracovat, to se deje proste automaticky.

ta podminka na while ma vypadat trochu jinak, zkus se jeste zamyslet.